packages/@okta/vuepress-theme-prose/components/Home.vue

```diff
@@ -74,16 +20,133 @@
+              src_dark: "/img/workforce2-dark.svg",
+              title: "Connect with APIs and SDKs",
+              text: "Build apps and services that interact directly with WIC for a completely integrated experience.",
+              list: [
+                {
+                  url: "https://developer.okta.com/docs/reference",
+                  text: "Explore reference APIs",
+                },
+                {
+                  url: "https://developer.okta.com/code",
+                  text: "Explore our SDKs ",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/embedded-authentication",
+                  text: "Explore embedded authentication use cases",
+                },
+              ]
+            },
+            {
+              src: "/img/workforce3-light.svg",
+              src_dark: "/img/workforce3-dark.svg",
+              title: "Authenticate",
+              text: "Define how your applications and APIs verify the identity of a user or device.",
+              list: [
+                {
+                  url: "https://developer.okta.com/docs/guides/redirect-authentication",
+                  text: "Start with redirect authentication",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/authenticators-overview/main",
+                  text: "Set up multifactor authentication",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/identity-providers",
+                  text: "Use an external IdP"
+                }
+              ]
+            },
+            {
+              src: "/img/workforce4-light.svg",
+              src_dark: "/img/workforce4-dark.svg",
+              title: "Brand and customize",
+              text: "Tailor your IAM tools with your organization’s brand and give users a consistent, familiar experience",
+              list: [
+                {
+                  url: "https://developer.okta.com/docs/guides/custom-url-domain/main",
+                  text: "Add a custom domain",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/custom-email/main",
+                  text: "Style email notifications",
+                },
+                {
+                  url: "https://developer.okta.com/docs/guides/custom-widget/main",
+                  text: "Customize the sign-in page"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          title: "Publish",
+          block: [
+            {
+              src: "/img/workforce5-light.svg",
+              src_dark: "/img/workforce5-dark.svg",
+              title: "Join the Okta Integration Network",
+              text: "Promote your OIDC, SAML, SCIM, or API service integration to thousands of customers and grow your business with the Okta Integration Network (OIN)",
```

Comment:
All the other items include a period. Either remove the period from those items, or add one here.